src/appmixer/wiz/core/UploadSecurityScan/inputs.vulnerabilityFindings.js

```
@@ -0,0 +1,176 @@
+    //             type: 'text',
+    //             index: 10,
+    //             label: 'Remediation',
+    //             required: true,
+    //             tooltip: 'The remediation for the vulnerability.'
+    //         },
+    //         validatedAtRuntime: {
+    //             type: 'toggle',
+    //             index: 11,
+    //             label: 'Validated At Runtime',
+    //             defaultValue: false,
+    //             tooltip: 'Indicates if the finding was detected during runtime (true), or if it was detected during offline or static scanning (false).'
```

ESLint

[error] This line has a comment length of 155. Maximum allowed is 150.

(max-len)